mybatis-plus/src/test/java/com/baeldung/mybatisplus/MyBatisPlusIntegrationTest.java

```diff
@@ -0,0 +1,206 @@
+package com.baeldung.mybatisplus;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import com.baeldung.mybatisplus.entity.Client;
+import com.baeldung.mybatisplus.idgenerator.TimestampIdGenerator;
+import com.baeldung.mybatisplus.service.ClientService;
+import com.baomidou.mybatisplus.core.conditions.query.LambdaQueryWrapper;
+import com.baomidou.mybatisplus.core.conditions.query.QueryWrapper;
+import com.baomidou.mybatisplus.core.conditions.update.LambdaUpdateWrapper;
+import com.baomidou.mybatisplus.core.toolkit.Wrappers;
+import com.baomidou.mybatisplus.extension.conditions.query.QueryChainWrapper;
+import com.baomidou.mybatisplus.extension.plugins.pagination.Page;
+
+@SpringBootTest
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+public class MyBatisPlusIntegrationTest {
+
+    @Autowired
+    private ClientService clientService;
+
+    @Autowired
+    private TimestampIdGenerator timestampIdGenerator;
+
+    @Test
+    @Order(1)
+    public void whenSaveIsInvoked_ThenClientIsInserted() {
```

Comment:
The format is given_when_then or given_when
```suggestion
    public void whenSaveIsInvoked_thenClientIsInserted() {
```

Reply:
fixed for all